### PR TITLE
Feature/create sign up page

### PR DESCRIPTION
**Pull request by eugenialcaraz**
#### Trello board reference:

* [#4](Create Sign Up Page)

---

#### Description:

* Add sign up page layout
* Adjust navigation through pages
* Email confirmation view & sign in after registration will be sent for approval in separated PR

---

#### Reviewers:

* @nicmalegre 
* @jpirazusta 
* @tobicorradi 
* @diego317 

---

#### Notes:

* Connection with the API is not done yet, there are no validations of the forms, just navigation through pages

---

#### Preview:

image

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -1,16 +1,16 @@
 import React from "react";
 import { Routes, Route } from "react-router-dom";
-import { SignIn, About } from "@components/index";
+import { About, SignIn, SignUp } from "@components/index";
 import { Landing, Pages } from "./pages";
-
 import { urlFormat } from "./utils/urlFormat";
```
Comment (nicmalegre) [inline src/App.tsx]: Could be '@/pages' and '@/utils' right? Like in `signIn`

**Review by nicmalegre — APPROVED** [review]
LGTM! Great work 🚀

File: src/App.tsx (hunk 1 above)
Comment (eugenialcaraz) [reply src/App.tsx]: True, fixed!

**Review by jpirazusta — APPROVED** [review]
👏

**Pull request merged**